src/control/security/config.go

```diff
@@ -20,17 +20,20 @@ import (
 )
 
 const (
-	certDir              = "/etc/daos/certs/"
-	defaultCACert        = certDir + "daosCA.crt"
-	defaultServerCert    = certDir + "server.crt"
-	defaultServerKey     = certDir + "server.key"
-	defaultAdminCert     = certDir + "admin.crt"
-	defaultAdminKey      = certDir + "admin.key"
-	defaultAgentCert     = certDir + "agent.crt"
-	defaultAgentKey      = certDir + "agent.key"
-	defaultClientCertDir = certDir + "clients"
-	defaultServer        = "server"
-	defaultInsecure      = false
+	certDir                    = "/etc/daos/certs/"
+	defaultCACert              = certDir + "daosCA.crt"
+	defaultServerCert          = certDir + "server.crt"
+	defaultServerKey           = certDir + "server.key"
+	defaultAdminCert           = certDir + "admin.crt"
+	defaultAdminKey            = certDir + "admin.key"
+	defaultAgentCert           = certDir + "agent.crt"
+	defaultAgentKey            = certDir + "agent.key"
+	defaultTelemetryServerCert = certDir + "telemetryserver.crt"
+	defaultTelemetryServerKey  = certDir + "telemetryserver.key"
```

Comment:
Minor naming suggestion - I think calling it "server" at this level may cause some confusion with the daos_server.
```suggestion
	defaultTelemetryServerCert = certDir + "telemetry.crt"
	defaultTelemetryServerKey  = certDir + "telemetry.key"
```